install.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+LISP=${LISP=sbcl}
```

Comment:
This is documented as `LANG`, but I prefer `LISP` because `LANG` is already used for locales.

Reply:
oops yea I messed up writing the documentation, fixed